taxcalc/decorators.py

```diff
@@ -283,7 +285,7 @@ def make_wrapper(func):
         try:
             jit_args = inspect.getargspec(jit).args + ['nopython']
         except TypeError:
-            print ("This should only be seen in RTD, if not install numba!")
+            #print ("This should only be seen in RTD, if not install numba!")
```

Comment:
@talumbau, this was printing repeatedly when building html (presumably when sphinx looked inside of each function in functions.py), which is a bit disconcerting/confusing. If this is ever seen outside of RTD, I presume Numba's nonexistence would have been noticed already. If we are worried about confusing error messages when packages are missing, should we check for the existence of dependencies explicitly?